zkevm-circuits/src/witness/block.rs

```diff
@@ -130,6 +135,108 @@ impl<F: Field> Block<F> {
     pub(crate) fn get_big_modexp(&self) -> Vec<BigModExp> {
         self.precompile_events.get_modexp_events()
     }
+
+    pub(crate) fn get_num_rows_required_no_padding(&self) -> usize {
```

Comment:
rename to print_evm_circuit_row_usage

Reply:
Renamed in commit [4aa4e81](https://github.com/scroll-tech/zkevm-circuits/pull/933/commits/4aa4e81892a065ce170436fb2d32115f70227570).